randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -587,9 +587,15 @@ Extra - map_name: rm_a5c14
               Mid-Air Morph (Beginner) or Power Grip Wall
               Speed Booster and Shinesparking Tricks (Beginner)
   > Pipe to The Tower
-      Any of the following:
-          Enabled Septogg Helpers or Hijump With Power Grip Wall
-          Speed Booster and Shinesparking Tricks (Beginner)
+      All of the following:
+          Any of the following:
+              # Climb room to pipe
```

Comment:
"Climb room to pipe" sounds weird. I assume you meant something like "Get to platform in front of pipe"